storage/src/main/java/com/google/modernstorage/storage/AndroidFileSystem.kt

```diff
@@ -337,12 +335,14 @@ class AndroidFileSystem(private val context: Context) : FileSystem() {
                 createdAtMillis = null,
                 lastModifiedAtMillis = lastModifiedTime,
                 lastAccessedAtMillis = null,
-                extras = mapOf(
-                    Path::class to path,
-                    Uri::class to uri,
-                    MetadataExtras.DisplayName::class to MetadataExtras.DisplayName(displayName),
-                    MetadataExtras.MimeType::class to MetadataExtras.MimeType(mimeType),
-                )
+                extras = buildMap {
+                    put(Path::class, path)
+                    put(Uri::class, uri)
+                    put(MetadataExtras.DisplayName::class, MetadataExtras.DisplayName(displayName))
+                    if (mimeType != null) {
+                        put(MetadataExtras.MimeType::class, MetadataExtras.MimeType(mimeType))
```

Comment:
This will be a breaking change for apps that always expect a mime type to be present, but maybe that's okay considering you've only done alpha releases so far?